<ui version="4.0" >
 <class>CMakeBuildSettings</class>
 <widget class="QWidget" name="CMakeBuildSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>537</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="KComboBox" name="buildDirs" />
     </item>
     <item>
      <widget class="QPushButton" name="addBuildDir" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeBuildDir" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Cache Values</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QTreeView" name="cacheList" >
        <property name="rootIsDecorated" >
         <bool>false</bool>
        </property>
        <property name="itemsExpandable" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Comment:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="commentText" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="showAdvanced" >
     <property name="text" >
      <string>Show Advanced</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="advancedBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Advanced</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>CMake Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KUrlRequester" name="kcfg_cmakeDir" />
      </item>
      <item>
       <widget class="QCheckBox" name="showInternal" >
        <property name="text" >
         <string>Show Advanced Values</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
